### PR TITLE
Add Django trove classifiers

### PR DESCRIPTION
**Pull request by lithammer**
Useful to see what the supported Django versions are.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -46,6 +46,13 @@ def read_md(filename):
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
+        'Framework :: Django',
+        'Framework :: Django :: 3',
+        'Framework :: Django :: 3.0',
+        'Framework :: Django :: 3.1',
+        'Framework :: Django :: 3.2',
+        'Framework :: Django :: 4',
+        'Framework :: Django :: 4.0',
```
Comment (lithammer) [inline setup.py:55]: I was't sure about the Django 2.x support, so I left that out.

Comment (millerdev) [reply setup.py:55]: Yes, Django 2.2 is supported, although I believe the EOL is tomorrow.

**Review by millerdev — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (lithammer) [reply setup.py:55]: Should I add it anyway?

Comment (millerdev) [reply setup.py:55]: That would be fine.

Comment (lithammer) [reply setup.py:57]: Done.

**Comment by millerdev** [comment]
@lithammer Thank you for contributing!

**Comment by lithammer** [comment]
On a slight related note, maybe the "Development Status" classifier should be updated? Doesn't feel like "alpha" software 😉

**Pull request merged**